templates/bot/js/workflow/README.md

```diff
@@ -68,7 +68,7 @@ The following files are project-related files. You generally will not need to cu
 
 # Customize your application
 
-By default a single command is generated that sends the `helloworldCommand.json` Adaptive Card when a user types `hello` in the private message chat with the bot.
```

Comment:
The workflow is initiated by typing a `helloWorld`.

Reply:
Fixed

Reply:
The message pattern is `helloWorld` rather than `hello` :)